app/controllers/planning_applications_controller.rb

```diff
@@ -264,6 +264,12 @@ def determine
     end
   end
 
+  def decision_notice
```

Comment:
We never actually had this action defined here which worked fine because Rails automatically render views with names that correspond to valid routes. Buuut that's probably bad practice and inconsistent